internal/os/os_test.go

```diff
@@ -0,0 +1,80 @@
+package os
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	root "github.com/equinix/metal-cli/internal/cli"
+	outputPkg "github.com/equinix/metal-cli/internal/outputs"
+	"github.com/spf13/cobra"
+)
+
+func TestCli_RegisterCommands(t *testing.T) {
+	subCommand := "operating-systems"
+	consumerToken := ""
+	apiURL := ""
+	Version := "metal"
+	rootClient := root.NewClient(consumerToken, apiURL, Version)
+	type fields struct {
+		MainCmd  *cobra.Command
+		Outputer outputPkg.Outputer
+	}
+	type args struct {
+		name string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    *cobra.Command
+		cmdFunc func(*testing.T, *cobra.Command)
+	}{
+		{
+			name: "NoArgs",
+			fields: fields{
+				MainCmd:  &cobra.Command{},
+				Outputer: outputPkg.Outputer(&outputPkg.Standard{}),
+			},
+			args: args{name: "testCommand"},
+			want: &cobra.Command{},
+			cmdFunc: func(t *testing.T, c *cobra.Command) {
+				t.Helper()
+				root := c.Root()
+				root.SetArgs([]string{subCommand})
+				if err := root.Execute(); err != nil {
+					t.Logf("%+v", root.Args)
+					t.Error("expected an error")
```

Comment:
I don't think we expect the CLI to error out if the `operating-systems` subcommand is called without arguments; it should successfully print the help text in that case.

I don't think we really need to add tests for running commands with no arguments for now--we should focus on commands that interact with the API--but I'm concerned that this test appears to be passing in CI.  What error is produced here?